remove libbitcoinconsensus

josibake opened the pull request:
file does not exist anymore, which leads to an error when using warcli build

josibake commented:
Note: it occurred to me this file might still be present for older releases, but seems fine to keep it in there vs writing some conditional logic based on the version for remove it?

pinheadmz commented:
Is this about the IsReachable patch or just cleanup to keep the image small?

willcl-ark commented:
It will be better to simply `rm -f` (don't fail if don't exist) them, no?

```bash
    && rm -f ${BITCOIN_PREFIX}/lib/libbitcoinconsensus.a \
    && rm -f ${BITCOIN_PREFIX}/lib/libbitcoinconsensus.so.0.0.0 \
```

josibake commented:
> Is this about the IsReachable patch or just cleanup to keep the image small?

Keeping the image small

willcl-ark commented:
> It will be better to simply `rm -f` (don't fail if don't exist) them, no?
> 
> ```shell
>     && rm -f ${BITCOIN_PREFIX}/lib/libbitcoinconsensus.a \
>     && rm -f ${BITCOIN_PREFIX}/lib/libbitcoinconsensus.so.0.0.0 \
> ```

I'd prefer this approach personally, as I don't think skipping the `strip` call will actually remove the libs? They will still be copied across (but be larger in size!)

josibake commented:
> I'd prefer this approach personally, as I don't think skipping the strip call will actually remove the libs?

Do you think its worth changing all of the calls to `rm` , instead of `strip`?

willcl-ark commented:
No?

rm deletes it, strip removes debug symbols

josibake committed:
remove libbitcoinconsensus

This file is no longer present in the repo. Change to rm -f so
that it is still removed when compiling an older PR/older version.

josibake commented:
@willcl-ark updated to use `rm -f` instead of strip

josibake commented:
CI failing due to https://github.com/docker/docker-py/issues/3256

willcl-ark commented:
> CI failing due to [docker/docker-py#3256](https://github.com/docker/docker-py/issues/3256)

Well thats silly!

willcl-ark commented:
One more rebase on current `main` in here?
